### ISSUE TITLE
1.5. Stochastic Gradient Descent（sklearn UsersGuide）に関してまとめる

### ISSUE THREAD

**Issue by takahiro-777** [takahiro-777/anaconda_tutorial#10]
http://scikit-learn.org/stable/modules/sgd.html
などが参考になるか
